src/packages/documents/documents/components/input-document/input-document.element.ts

```diff
@@ -63,8 +63,8 @@ export class UmbInputDocumentElement extends FormControlMixin(UmbLitElement) {
 	@property({ type: String })
 	startNodeId?: string;
 
-	@property({ type: String })
-	filter?: string;
+	@property({ type: Array })
+	filter?: string[] | undefined;
```

Comment:
I know this is not new in this PR. But there is something about the name here that I'm not fond of.

Could it be named `acceptableContentTypeIds`, to make it more explicit what it does?

Same goes for the others of this PR.

Reply:
@nielslyngsoe I've replaced `filter` with `allowedContentTypeIds`, in commit 0897acee74fd61a137f29ba602f2257919a572ff.